packages/circuit-ui/components/Table/components/TableRow/TableRow.tsx

```diff
@@ -64,20 +71,7 @@ const clickableStyles = ({ theme, onClick }) =>
 
 const Tr = styled.tr(baseStyles, clickableStyles);
 
-const TableRow = ({ onClick, ...props }) => {
-  const handleKeyDown = (event) => {
-    if (isEnter(event) || isSpacebar(event)) {
-      onClick();
-    }
-  };
```

Comment:
I removed this entirely when I noticed that pressing enter or the spacebar on the focused element already triggers onClick by default. I left the test that covers this in the spec for sanity's sake

Reply:
\*ahem\* good point, will do 🙈

Reply:
Verified on the [latest deploy preview](https://oss-circuit-ui-4vqd2duc6-sumup.vercel.app/?path=/story/components-table--base) in Chrome, Firefox and Safari, looks good 👍